### ISSUE TITLE
in core dbm

### ISSUE THREAD

**Issue by Aokromes** [TrinityCore/TrinityCoreCustomChanges#29]
**Description:**

maybe a good idea can be a way to enable on screen warnings and co for people without addons.

**TC rev. hash/commit:** 

4c0a8ad3b3f9

**TDB version:**  

N/A

**Operating system:** 

N/A

**Comment by Keader** [comment]
dont think that worth it...
because will need adapt every single script existing to emmit alerts.

**Comment by Aokromes** [comment]
fill scripts of debug logs and enable them if setting enabled :P

**Comment by Keader** [comment]
exactly... for?
we dont have people to fix those scripts, but we have people to flood with unneeded code to people that cant install a addon? o.o

**Comment by Aokromes** [comment]
> exactly... for?
> we dont have people to fix those scripts, but we have people to flood with unneeded code to people that cant install a addon? o.o

well this is "custom" not main project :)

**Comment by Keader** [comment]
yes but do you understand that is higher cost for little benefit?

**Comment by offl** [comment]
You can add a message to alarm about spell cast directly in core and use it if creature is a boss. Filter serverside spells and spells with certain attributes of course. You can make message appear in any color also
You can also add message to inform about switching phase if boss uses EnterPhase function or something like that
This way you don't need to edit any script

You can also create config to exclude spells you don't want to alarm about. Actually I think this may lead to spam and people will ask you to remove it. It will be used for all players and it looks really bad imho

But I really doubt there is at least one person who wants to add messages directly to all scripts, phases and spells and update them after every update in /scripts
